Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1163,7 +1189,7 @@ set_option linter.deprecated false in
 theorem lt_sup {ι : Type u} {f : ι → Ordinal.{max u v}} {a} : a < sup.{_, v} f ↔ ∃ i, a < f i := by
   simpa only [not_forall, not_le] using not_congr (@sup_le_iff.{_, v} _ f a)
 
-theorem ne_iSup_iff_lt_iSup {ι : Type u} {f : ι → Ordinal.{max u v}} :
+theorem ne_iSup_iff_lt_iSup {ι} {f : ι → Ordinal.{u}} [Small.{u} ι] :
```

Comment:
We should probably be deprecating, rather than updating, this lemma. See [this](https://github.com/leanprover-community/mathlib4/pull/15820#discussion_r1740447705) comment by Yaël.

Reply:
I've gone ahead and done this at #16997.

Reply:
Is it alright if I add that PR as a dependence for this one? I'm hoping it can get approved quickly.